main.go

```diff
@@ -59,6 +45,38 @@ func update() {
 	fmt.Println("Updated Successfully.")
 }
 
+func parseJSON(s string) {
```

Comment:
Rename `s` to `filePath` or something. The variable names should be as explicit as possible to avoid confusion.